app/src/main/java/com/group1/movebetter/view_model/controller/BirdController.kt

```diff
@@ -6,14 +6,13 @@ import com.group1.movebetter.model.*
 import com.group1.movebetter.repository.Repository
 import com.mapbox.geojson.Feature
 import com.mapbox.geojson.Point
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.launch
+import kotlinx.coroutines.*
 
 class BirdController(private val viewModelScope: CoroutineScope, private val repository: Repository, private val mapController: MapController) {
 
     // TODO: retrieve the tokens from a database
-    private val refresh: Token = Tokens.refresh
-    private val access: Token = Tokens.access
+    //private val refresh: Token = Tokens.refresh
+    //private val access: Token = Tokens.access
```

Comment:
Die beiden Zeilen sowie die Tokens.kt können nun gelöscht werden (ebenfalls der Eintrag in der .gitignore).